src/Components/test/testassets/Components.TestServer/RazorComponents/FormWithDefaultContextApp.razor

```diff
@@ -1,26 +0,0 @@
-﻿@using Components.TestServer.RazorComponents.Pages.Forms
-@using Components.TestServer.RazorComponents.Shared
-
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="utf-8" />
-    <base href="/subdir/" />
-    <HeadOutlet />
-</head>
-<body>
-    <Router AppAssembly="@typeof(FormWithDefaultContextApp).Assembly">
-        <Found Context="routeData">
-            <RouteView RouteData="@routeData" DefaultLayout="@typeof(FormsLayout)" />
-            <FocusOnNavigate RouteData="@routeData" Selector="h1" />
-        </Found>
-        <NotFound>
-            <PageTitle>Not found</PageTitle>
-            <LayoutView>
-                <p role="alert">Sorry, there's nothing at this address.</p>
-            </LayoutView>
-        </NotFound>
-    </Router>
-    <script src="_framework/blazor.web.js" suppress-error="BL9992"></script>
-</body>
-</html>
```

Comment:
Is this not used?

Reply:
It's not a big deal, but this was only used to vary the layout, so I added a `_Imports.razor` to set the layout for forms test pages instead. It's only so we can keep the E2E tests runnable manually (by launching `TestServer`), which is much easier when they have a common root component.

For cases where it's completely necessary to vary the root component I guess we have no choice, and then would have to add some new entry to the list of test apps, but whenever we can avoid growing that it will be easier to find the test case you want.